client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -1,60 +1,59 @@
 /**
  * External dependencies
  */
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect } from 'react';
 import { useI18n } from '@automattic/react-i18n';
+import { Icon } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import AnimatedPlaceholder from '../animated-placeholder';
 import CreateAndRedirect from './create-and-redirect';
 import { useNewQueryParam } from '../../path';
 import './style.scss';
+/**
+ * Internal dependencies
+ */
+import { ProgressBar } from '@automattic/components';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 const CreateSite: FunctionComponent< {} > = () => {
 	const { __: NO__ } = useI18n();
 	const shouldTriggerCreate = useNewQueryParam();
+	const progressTotal = 100;
+	const progressStepCount = 3;
+	const [ progressComplete, setProgressComplete ] = React.useState( 0 );
+	const [ shouldCreateAndRedirect, setCreateAndRedirect ] = React.useState( false );
 
-	const createAndRedirect = shouldTriggerCreate ? <CreateAndRedirect /> : null;
+	useEffect( () => {
+		if ( progressComplete < progressTotal ) {
+			setTimeout( () => {
+				const progressFactor = progressTotal / progressStepCount;
+				setProgressComplete( progressComplete + progressFactor );
+			}, 2000 );
+		}
+
+		if ( progressComplete >= progressTotal ) {
+			setCreateAndRedirect( true );
+		}
+	}, [ progressComplete, shouldTriggerCreate ] );
 
 	return (
 		<div className="create-site__background">
-			{ createAndRedirect }
+			{ shouldCreateAndRedirect ? <CreateAndRedirect /> : null }
```

Comment:
not; or `{ shouldCreateAndRedirect && <CreateAndRedirect /> }`

Reply:
Exactly that 👍